dotnet/src/SemanticKernel.Core/KernelExtensions.cs

```diff
@@ -514,12 +514,14 @@ public static IKernelBuilderPlugins AddFromObject(this IKernelBuilderPlugins plu
     /// <exception cref="ArgumentException"><paramref name="promptTemplate"/> is empty or composed entirely of whitespace.</exception>
     /// <exception cref="KernelFunction">The function failed to invoke successfully.</exception>
     /// <exception cref="KernelFunctionCanceledException">The <see cref="KernelFunction"/>'s invocation was canceled.</exception>
+    /// <param name="cancellationToken">The cancellation token.</param>
```

Comment:
Can you use the same comment as on
https://github.com/microsoft/semantic-kernel/pull/4298/files#diff-1966179877500641f0ce2da0026cb59eecdd711eccab3d58e53b0aaca6ba05efL546